reddit_decider/__init__.py

```diff
@@ -197,6 +198,20 @@ def _clear_identifiers_and_set(self, identifier: str, identifier_type: Literal[I
         ctx[identifier_type] = identifier
         return ctx
 
+    def _format_decision(self, decision_dict: Dict[str, str]):
+        out = {}
+        # cast id to int
+        for k, v in decision_dict.items():
+            if k == "id":
+                try:
+                    out[k] = int(v)
```

Comment:
@julieahnt added casting id to int because it's quite difficult to have a HashMap in rust with values of different types.